src/OpenRiaServices.Client.Web/Framework/OpenRiaServices.Client.Web.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net472;net6.0;netstandard2.0;</TargetFrameworks>
+    <TargetFrameworks>net472;net6.0;netstandard2.0;net8.0;</TargetFrameworks>
```

Comment:
I think we can delay with adding the net8 compile target for this project if we don't need to make any other modifications to the code or project.

Otherwise we should update the .nuspec file to include the compiled code

Reply:
This is needed for T4CodeGen and some Tools codegen tests when loading project in MsBuildHelper. Changing the LoadProject method in MsBuildHelper to use net6 as framework makes all but 1 test succeed, this test, CodeGen_CustomAttrGen_CtrSelectorTest,  fails at compiling the generated assembly because of the net6/net8 mismatch

Reply:
It sounds wrong, but does not needs to be fixed now. We can keep it